README.md

```diff
@@ -87,9 +87,18 @@ https://github.com/openebs/zfs-localpv/blob/master/docs/faq.md#6-how-to-add-cust
 OpenEBS ZFS driver components can be installed by running the
 following command.
 
+Ubuntu
```

Comment:
@jlcox1970 now that PR https://github.com/openebs/zfs-localpv/pull/204 has been merged. Can we add CentOS 7 and 8 in the list here to use same yaml.

Reply:
I believe that this PR is no longer required as the yaml is no agnostic to the OS 
@pawanpraka1 please cancel/delete this PR

Reply:
that is correct, but we run the e2e test cases on ubuntu and centos only (https://github.com/openebs/zfs-localpv/pull/203/files#diff-04c6e90faac2675aa89e2176d2eec7d8L35). For the other platforms there might be some glitches. Thinking how can we convey that message :thinking: